File: sample-app-java/app/src/main/java/com/bluejeans/android/sdksample/participantlist/ParticipantChatItemListener.java
package com.bluejeans.android.sdksample.participantlist;

import android.widget.TextView;

import com.bluejeans.bluejeanssdk.meeting.ParticipantsService;

public interface ParticipantChatItemListener {
    void onMessageCountChange(ParticipantsService.Participant participant, TextView mUnreadCount);

    void onParticipantClick(ParticipantsService.Participant participant);
}

File: sample-app-kotlin/app/src/main/java/com/bluejeans/android/sdksample/participantlist/ParticipantChatItemListener.kt
package com.bluejeans.android.sdksample.participantlist

import android.widget.TextView
import com.bluejeans.bluejeanssdk.meeting.ParticipantsService

interface ParticipantChatItemListener {
    fun onMessageCountChange(participant: ParticipantsService.Participant, mUnreadCount: TextView)
    fun onParticipantClick(participant: ParticipantsService.Participant)
}